best-practices/tagging/index.html

```diff
@@ -601,66 +631,94 @@ <h4>Tables</h4>
-    &lt;th>⠠⠎⠉⠕⠗⠑&lt;/th>
-  &lt;/tr>
-  &lt;tr>
-    &lt;td>⠠⠚⠕⠓⠝&lt;/td>
-    &lt;td>⠼⠃⠑&lt;/td>
-  &lt;/tr>
-  &lt;tr>
-    &lt;td>⠠⠍⠁⠭&lt;/td>
-    &lt;td>⠼⠃⠃&lt;/td>
-  &lt;/tr>
-  &lt;tr>
-    &lt;td>⠠⠞⠁⠗⠁&lt;/td>
-    &lt;td>⠼⠃⠓&lt;/td>
-  &lt;/tr>
+	&lt;tr>
+		&lt;th id="name-header" scope="col">⠠⠝⠁⠍⠑&lt;/th>
+		&lt;th id="score-header" scope="col">⠠⠎⠉⠕⠗⠑&lt;/th>
+	&lt;/tr>
+	&lt;tr>
+		&lt;td headers="name-header">⠠⠚⠕⠓⠝&lt;/td>
+		&lt;td headers="score-header">⠼⠃⠑&lt;/td>
+	&lt;/tr>
+	&lt;tr>
+		&lt;td headers="name-header">⠠⠍⠁⠭&lt;/td>
+		&lt;td headers="score-header">⠼⠃⠃&lt;/td>
+	&lt;/tr>
+	&lt;tr>
+		&lt;td headers="name-header">⠠⠞⠁⠗⠁&lt;/td>
+		&lt;td headers="score-header">⠼⠃⠓&lt;/td>
+	&lt;/tr>
 &lt;/table></code></pre>
 				</aside>
 				
 				<aside class="example" title="A table with a recommended table type">
 					<pre><code class="html">&lt;table class="listed">
-  &lt;tr>
-    &lt;th>⠠⠝⠁⠍⠑&lt;/th>
-    &lt;th>⠠⠎⠉⠕⠗⠑&lt;/th>
-  &lt;/tr>
-  &lt;tr>
-    &lt;td>⠠⠚⠕⠓⠝&lt;/td>
-    &lt;td>⠼⠃⠑&lt;/td>
-  &lt;/tr>
-  &lt;tr>
-    &lt;td>⠠⠍⠁⠭&lt;/td>
-    &lt;td>⠼⠃⠃&lt;/td>
-  &lt;/tr>
-  &lt;tr>
-    &lt;td>⠠⠞⠁⠗⠁&lt;/td>
-    &lt;td>⠼⠃⠓&lt;/td>
-  &lt;/tr>
+	&lt;tr>
+		&lt;th id="name-header" scope="col">⠠⠝⠁⠍⠑&lt;/th>
+		&lt;th id="score-header" scope="col">⠠⠎⠉⠕⠗⠑&lt;/th>
+	&lt;/tr>
+	&lt;tr>
+		&lt;td headers="name-header">⠠⠚⠕⠓⠝&lt;/td>
+		&lt;td headers="score-header">⠼⠃⠑&lt;/td>
+	&lt;/tr>
+	&lt;tr>
+		&lt;td headers="name-header">⠠⠍⠁⠭&lt;/td>
+		&lt;td headers="score-header">⠼⠃⠃&lt;/td>
+	&lt;/tr>
+	&lt;tr>
+		&lt;td headers="name-header">⠠⠞⠁⠗⠁&lt;/td>
+		&lt;td headers="score-header">⠼⠃⠓&lt;/td>
+	&lt;/tr>
 &lt;/table></code></pre>
 				</aside>
 				<aside class="example" title="A table with header and body">
 					<pre><code class="html">&lt;table>
-  &lt;thead>
-    &lt;tr>
-      &lt;th>⠠⠙⠁⠞⠑&lt;/th>
-      &lt;th>⠠⠙⠑⠎⠉⠗⠊⠏⠞⠊⠕⠝&lt;/th>
-      &lt;th>⠠⠉⠗⠑⠙⠊⠞&lt;/th>
-      &lt;th>⠠⠙⠑⠃⠊⠞&lt;/th>
-    &lt;/tr>
-  &lt;/thead>
-  &lt;tbody>
-    &lt;tr>
-      &lt;td>⠼⠛ ⠠⠚⠥⠝&lt;/td>
-      &lt;td>⠠⠉⠁⠎⠓ ⠺⠊⠞⠓⠙⠗⠁⠺⠁⠇&lt;/td>
-      &lt;td>&lt;/td>
-      &lt;td>⠈⠎⠼⠁⠚⠚⠲⠚⠚&lt;/td>
-    &lt;/tr>
-    &#8230;
-  &lt;/tbody>
+	&lt;thead>
+		&lt;tr>
+		&lt;th id="date-header" scope="col">⠠⠙⠁⠞⠑&lt;/th>
+		&lt;th id="description-header" scope="col">⠠⠙⠑⠎⠉⠗⠊⠏⠞⠊⠕⠝&lt;/th>
+		&lt;th id="credit-header" scope="col">⠠⠉⠗⠑⠙⠊⠞&lt;/th>
+		&lt;th id="debit-header" scope="col">⠠⠙⠑⠃⠊⠞&lt;/th>
+		&lt;/tr>
+	&lt;/thead>
+	&lt;tbody>
+		&lt;tr>
+		&lt;td headers="date-header">⠼⠛ ⠠⠚⠥⠝&lt;/td>
+		&lt;td headers="description-header">⠠⠉⠁⠎⠓ ⠺⠊⠞⠓⠙⠗⠁⠺⠁⠇&lt;/td>
+		&lt;td headers="credit-header">&lt;/td>
+		&lt;td headers="debit-header">⠈⠎⠼⠁⠚⠚⠲⠚⠚&lt;/td>
+		&lt;/tr>
+		&#8230;
+	&lt;/tbody>
 &lt;/table></code></pre>
 				</aside>
+				<aside class="example" title="A table with long headings that may need to be abbreviated">
+					<pre><code class="html">&lt;table>
+	&lt;thead>
+		&lt;tr>
+		&lt;th id="fruit-header" abbr="Fruit" scope="col">⠠⠋⠗⠥⠊⠞&lt;/th>
```

Comment:
Should the abbr value also be in braille?

Reply:
Yes, I think so too.